mappings/net/minecraft/world/gen/chunk/ChunkGenerator.mapping

```diff
@@ -141,6 +142,8 @@ CLASS net/minecraft/class_2794 net/minecraft/world/gen/chunk/ChunkGenerator
 	METHOD method_40450 getDebugHudText (Ljava/util/List;Lnet/minecraft/class_7138;Lnet/minecraft/class_2338;)V
 		ARG 1 text
 	METHOD method_41039 streamStructureSets ()Ljava/util/stream/Stream;
+	METHOD method_41042 getStructuresRegistryForCodec (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/Products$P1;
```

Comment:
Since this returns a record codec builder that defines how to get a structure set registry from the chunk generator,
```suggestion
	METHOD method_41042 createStructureSetRegistryGetter (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/Products$P1;
```

Reply:
yeah, I like that better. I wasn't 100% on this one for sure.